docker/auth/realm.json

```diff
@@ -56,12 +56,34 @@
       "realmRoles": ["user"]
     },
     {
-      "username": "user-e2e",
+      "username": "user-e2e-chromium",
       "enabled": true,
       "credentials": [
         {
           "type": "password",
-          "value": "password-e2e"
+          "value": "password-e2e-chromium"
+        }
+      ],
+      "realmRoles": ["user"]
+    },
+    {
+      "username": "user-e2e-webkit",
+      "enabled": true,
+      "credentials": [
+        {
+          "type": "password",
+          "value": "password-e2e-webkit"
+        }
+      ],
+      "realmRoles": ["user"]
+    },
+    {
+      "username": "user-e2e-firefox",
+      "enabled": true,
+      "credentials": [
+        {
+          "type": "password",
+          "value": "password-e2e-firefox"
```

Comment:
Where is the logic binding the username with the tested browser?

Reply:
or just to add the name of the browser while creating the team ? to ensure team's unicity?

Reply:
I do it already, but it is still prone to flakiness.
Lets see:  
- chromium create 40 entries.
- my firefox tests start for any reason only after these 40 chromium entries 
- firefox tests the order, it creates 3 entries
- my 3 entries are displayed - we are ok
- firefox test click on the button sort
- The 20 oldest entries are displayed - they are the chromium ones
- firefox cannot assert that his 3 entries are sort correctly